### PR TITLE
Publish IPIPs to specs.ipfs.tech

### PR DESCRIPTION
**Pull request by hacdias**
Based on #394.

A preview of what moving the IPIPs can look like. Nice things to get:

- [x] Rebase and update once https://github.com/ipfs/specs/pull/408 is merged.
- [x] Merge https://github.com/ipfs/specs/pull/394
- [x] "Related Issues" on the frontmatter, such that it is displayed in a similar way to the editors (depends on https://github.com/ipfs/spec-generator/pull/6)
- [x] Separate styling, maybe a different accent colour: yellow? orange? We should use something we already have.
- [x] Add IPIPs link in the main index.
- [x] Update the IPIP process to include right paths.
- [x] Decide on URL structure (done in https://github.com/ipfs/specs/pull/385#discussion_r1143876565)
- [x] Simplified listings by using a common include with the list. This way they don't get out of sync (thanks @darobin)

### CODE REVIEW

**Comment by hacdias** [comment]
IPIP styling idea:

screenshot

File: src/index.html
Comment (darobin) [inline src/index.html:73]: Wouldn't it be better to have this as an include shared with the index of `/http-gateways/`? Otherwise these are going to go out of sync.

File: src/index.html
Comment (darobin) [inline src/index.html:123]: It might be a good idea to also list IPIPs?

File: src/ipips/index.html
Comment (darobin) [inline src/ipips/index.html:13]: I think that this might be better in meta, no?

File: src/ipips/ipip-0001.md
Comment (darobin) [inline src/ipips/ipip-0001.md:12]: Does it help if I take that on in the generator?

File: src/ipips/ipip-0002.md
Comment (darobin) [inline src/ipips/ipip-0002.md:88]: Was there a legal review on picking CC0? I ask because I know that the WHATWG switched from CC0 to CCBY4. I forget what the reason was but I can look it up.

**Review by darobin — COMMENTED** [review]
These are mostly questions and general comments, nothing on the critical path.

File: src/index.html
Comment (hacdias) [reply src/index.html]: Good idea

File: src/ipips/ipip-0001.md
Comment (hacdias) [reply src/ipips/ipip-0001.md:12]: Made a PR: https://github.com/ipfs/spec-generator/pull/6, would be great to get it merged and released!

**Comment by hacdias** [comment]
I marked it as ready for review as I think most things are "ready". We just have to merge https://github.com/ipfs/spec-generator/pull/6 and settle on how we want to style the IPIPs.

File: src/ipips/ipip-0002.md
Comment (hacdias) [reply src/ipips/ipip-0002.md:86]: cc @lidel

**Comment by hacdias** [comment]
This is how IPIPs look now. Just have to add a `maturity: proposal`:

bafybeid3kgm7r5slbo6tgvqegp2j4zyjdgid2oabzoxx25bld55xnnp32m on fleek co_ipips_ipip-0337_

File: src/ipips/ipip-0002.md
Comment (lidel) [reply src/ipips/ipip-0002.md:87]: @darobin there was no legal review, intention was to "lock specs open" in the spirit of  https://protocol.ai/blog/announcing-the-permissive-license-stack/
if you could find out the reason for avoiding CC0 that would be really helpful (better to switch sooner than later)

File: src/ipips/ipip-0002.md
Comment (lidel) [inline src/ipips/ipip-0002.md:3]: :thought_balloon: having rattified and shipped proposals marked as "proposal" feels weird.

@hacdias  @darobin  thoughts on adding second status for IPIPs named  'ratified' and have analogous watermark?
That was it would be visually clear when a proposal vs ratified spec change description is displayed.

**Review by lidel — COMMENTED** [review]
@hacdias @darobin do either of you needs this to land this week before Governance track and related talks?
If so, feel free to merge and we can fix later.
If not critical, let's keep this open, and get back to it after we are done with IPFS Thing.

**Comment by hacdias** [comment]
@lidel not critical for our presentation, so we can push it back for next week.

File: src/_includes/specs/ipips.html
Comment (hacdias) [reply src/_includes/specs/ipips.html:7]: We can, but we should do it after: https://github.com/ipfs/spec-generator/issues/11.

File: src/ipips/ipip-0002.md
Comment (hacdias) [reply src/ipips/ipip-0002.md:3]: Perhaps we can use a completely different tag altogether. I had to add `if`s in the spec-generator since this maturity is handled differently from the remaining ones. What about: `ipip: proposal` and `ipip: ratified`? I

Comment (hacdias) [reply src/ipips/ipip-0002.md]: I added the aforementioned options.

File: src/ipips/ipip-0002.md
Comment (darobin) [reply src/ipips/ipip-0002.md:87]: I don't think that this should block this update — I'll make a separate pass on doc license in genera.

File: src/css/specs.css
Comment (darobin) [inline src/css/specs.css:32]: Couldn't this use `background-image` on `body`? It would be less hacky.

**Review by darobin — APPROVED** [review]
It would be nice to do the watermark differently, but I would rather ship now and fix later than hold up just for that.

**Comment by hacdias** [comment]
@darobin i don't like the way I did the backgrounds either. Suggestions?

**Review by lidel — APPROVED** [review]
lgtm, we can fix cosmethics in follow-up.

Merging to unblock IPIP work in #402  using the new template

**Pull request merged**
